tables/automl/notebooks/retail_product_stockout_prediction/README.md

```diff
@@ -0,0 +1,371 @@
+
+6. The imported dataset is now analyzed \
+This helps you analyze the size of your dataset, dig into missing values if any, calculate correlation, mean and standard deviation. If this data quality looks good to you then we can move on to the next tab i.e. train. \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%201%202019-03-25%20at%2012.43.13%20AM.png" width="50%" height="50%">\
+
+7. Train \
+    a. Select a model name \
+    b. Select the training budget \
+    c. Select all features you would like to use for training \
+    d. Select optimization objectives. Such as: ROC, Log Loss or PR curve \
+    (As our data is imbalances we use PR curve as our optimization metric) \
+    e. Click TRAIN \
+    f. Training the model can take some time \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%208%202019-03-13%20at%201.34.08%20PM.png" width="50%" height="50%">\
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%202%202019-03-25%20at%2012.44.18%20AM.png" width="50%" height="50%">\
+
+8. Once the model is trained you can click on the evaluate tab \
+This tab gives you stats for model evaluation  \
+         For example our model shows  \
+         Area Under Precision Recall Curve: 0.645  \
+         Area Under ROC Curve: 0.893 \
+         Accuracy: 92.5% \
+         Log Loss: 0.217 \
+Selecting the threshold lets you set a desired precision and recall on your predictions. \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%203%202019-03-25%20at%2012.49.40%20AM.png" width="50%" height="50%">\
+
+9. Using the model created let's use batch prediction to predict stock-out \
+    a. Batch prediction data inputs could come from BigQuery or your GCS bucket. \
+    b. Select the GCS bucket to store the results of your batch prediction \
+    c. Click Send Batch Predictions \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%2012%202019-03-13%20at%201.56.43%20PM.png" width="50%" height="50%">\
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%2013%202019-03-13%20at%201.59.18%20PM.png" width="50%" height="50%">\
+
+
+## Building the model using AutoML Tables Python Client Library
+
+In this notebook, you will learn how to build the same model as you have done on the AutoML Tables UI using its Python Client Library.
+
+
+## Evaluation results and business impact
+
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%203%202019-03-25%20at%2012.49.40%20AM.png" width="50%" height="50%">\
```

Comment:
remove `< img src="` and replace with `![alt text](` 
and replace  `width="50%"` height="50%">` with `)`